src/nodes/FunctionTypeName.js

```diff
@@ -24,16 +24,19 @@ const stateMutability = (node) =>
     : '';
 
 export const FunctionTypeName = {
-  print: ({ node, path, print }) => [
-    'function(',
-    printSeparatedList(path.map(print, 'parameterTypes')),
-    ')',
-    indent(
-      group([
-        visibility(node),
-        stateMutability(node),
-        returnTypes(node, path, print)
-      ])
-    )
-  ]
+  print: ({ node, path, print }) =>
```

Comment:
FunctionTypeName did not follow the same rules we struggled for so long regarding how a function declaration should group and indent.
Here we are picking up this case we forgot.